src/ansys_sphinx_theme/__init__.py

```diff
@@ -538,6 +538,83 @@ def check_for_depreciated_theme_options(app: Sphinx):
         )
 
 
+def extract_whatsnew(app, doctree, docname):
+    """Extract the what's new content from the document."""
+    config_options = app.config.html_theme_options
+
+    whats_new_options = config_options.get("whatsnew")
+    if not whats_new_options:
+        return
+    no_of_contents = whats_new_options.get("no_of_headers", 3)
+    document_name = whats_new_options.get("file", "release-note")
+    get_doctree = app.env.get_doctree(document_name)
+    whats_new_content = []
```

Comment:
Agree on a naming convention. Either `whats_new` or `whatsnew`. Also, these are used later. If possible, instantiate the variables right before using them. If they are used for storing the values originated in a loop, then instantiate it right before the loop.